buddyboss/buddyboss-platform
# Fixed - Text formatting option is overlapping with the media uploader

### All Submissions:


Fixes #839 

### How to test the changes in this Pull Request:

1. Go to the news feed
2. Click on the media uploader icon
3. Click on the text formatting option
4. Notice the overlapping UI

### Proof Screenshots or Video

https://prnt.sc/sco9va

### Other information:
